FSharpBuild.Directory.Build.props

```diff
@@ -79,6 +79,10 @@
     <DefineConstants>$(DefineConstants);TESTING_ON_LINUX</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)' == 'Proto'">
```

Comment:
Don,
this isn't necessary.  The build failed because the StateMachinesHelpers module is marked RequiresPreview
https://github.com/dotnet/fsharp/blob/main/src/FSharp.Core/resumable.fsi#L69
````
/// Contains functions for composing resumable code blocks
[<Microsoft.FSharp.Core.Experimental("Experimental library feature, requires '--langversion:preview'")>]
[<Microsoft.FSharp.Core.RequireQualifiedAccess>]
module ResumableCode =
````

Since ResumableStateMachines is a finished feature we can remove this attribute and the  COMPILING_PROTO hack 
stuff should build perfect.
https://github.com/dotnet/fsharp/blob/main/src/Compiler/Facilities/LanguageFeatures.fs#L98
````
                LanguageFeature.ResumableStateMachines, languageVersion60
````

Reply:
thanks